### PR TITLE
Add angular-base-apps

### PR DESCRIPTION
**Pull request by soumak77**

### CODE REVIEW

**Comment by soumak77** [comment]
Is there anything I'm missing before this PR can get merged?

File: files/angular-base-apps/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "angular-base-apps",
+  "repo": "base-apps/angular-base-apps",
+  "files": {
+    "include": ["base-apps.css", "base-apps.min.css", "base-apps.js", "base-apps.min.js", "base-apps-templates.js", "base-apps-templates.min.js"]
```
Comment (jimaek) [inline files/angular-base-apps/update.json]: This way you are including the files from the / directory. https://github.com/base-apps/angular-base-apps
Which doesnt have them. You need to set the correct paths relative do your file structure in your repo.

Comment (soumak77) [inline files/angular-base-apps/update.json]: fixed

File: files/angular-base-apps/update.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "angular-base-apps",
+  "repo": "base-apps/angular-base-apps",
+  "files": {
+    "include": ["dist/css/base-apps.css", "dist/css/base-apps.min.css", "dist/js/base-apps.js", "dist/js/base-apps.min.js", "dist/js/base-apps-templates.js", "dist/js/base-apps-templates.min.js"]
```
Comment (jimaek) [inline files/angular-base-apps/update.json]: The end result of this config will be that all files are located in the same directory. Without css/ js/. Is this correct?

Comment (soumak77) [inline files/angular-base-apps/update.json]: Yes, though if I were to change that in the future, how would I do that?

Comment (jimaek) [inline files/angular-base-apps/update.json]: You would need to set `basePath` to dist/ and then change the `include` to `./css/base-apps.css` to retain the directory structure.

**Comment by jimaek** [comment]
Should I merge?

**Comment by soumak77** [comment]
Yes please, thanks.  The directory structure is correct for now.

**Pull request merged**